### ISSUE TITLE
Data entry part for Analysis of EPI2ME 16S CSV Output

### ISSUE THREAD

**Issue by xuanbella** [epi2me-labs/tutorials#9]
Hi,

Thanks for developing the pipeline for data analysis. They are quite helpful. I have used the EPI2ME 16S CSV Output pipeline before, when the "Use your own data" part was made of two cells where we can copy and paste the path there. That was quite clear to use. While for current version, the data entry part was mentioned as "If you wish to analyse your own data rather than the sample data, you can edit the value .fastq input variable below." However, i didnt fine any variable name as ".fastq" there. Would you please help clarify this?

Thanks!

**Comment by cjw85** [comment]
Hi @xuanbella. The explanatory text is erroneous (its some boilerplate text that hasn't been customised to the 16S notebook). We will update that.

You need to simply fill in a filepath to the CSV file downloaded from the EPI2ME in the form:
![image](https://user-images.githubusercontent.com/3592009/97155177-dc02a600-176c-11eb-9d2d-edfd9590b08f.png)
and then press `Enter`.

**Comment by xuanbella** [comment]
> Hi @xuanbella. The explanatory text is erroneous (its some boilerplate text that hasn't been customised to the 16S notebook). We will update that.
> 
> You need to simply fill in a filepath to the CSV file downloaded from the EPI2ME in the form:
> ![image](https://user-images.githubusercontent.com/3592009/97155177-dc02a600-176c-11eb-9d2d-edfd9590b08f.png)
> and then press `Enter`.

Hi,

Thanks for getting back to me. For this data entry dialog as the photo you attached, actually it is not shown in current work flow. Instead, there are a couple of coding line there for data entry in both github page and colab page. Would you please help check with this? Many thanks.

**Comment by cjw85** [comment]
With the latest versions of the notebooks we are now recommending that people use the default JupyterLab interface rather than Google Colab. The EPI2ME Labs Launcher has been amended to reflect this if you are using the newest version:

![image](https://user-images.githubusercontent.com/3592009/97430539-ef4e7680-1910-11eb-8b61-817d5121ea64.png)

The form elements will display correctly if you follow the "Open EPI2ME Labs" link in the Launcher and navigate to the 16S notebook from the start page.

**Comment by xuanbella** [comment]
Hi, 
Thanks for clarification. I have upgraded my EPI2ME labs launcher and accessed the notebook. While, still get the errors as below. 

![Screenshot from 2020-10-29 10-40-45](https://user-images.githubusercontent.com/69881217/97508628-33567f80-19d4-11eb-993e-31bd7ce4018c.png)

**Comment by cjw85** [comment]
Hi @xuanbella,

I've have fixed some small errors in the notebook. It won't be immediately available within EPI2MELabs, but you download the file from here: https://github.com/epi2me-labs/tutorials/blob/master/Analysis_of_EPI2ME_16S_CSV_Output.ipynb

**Comment by xuanbella** [comment]
Hi Chris,
Thanks for update the notebook. It worked with my 16SCSV file perfectly. However while i was using WIMP CSV file, I got the error as the figure. I noticed that the CSV files we downloaded fro 16S and WIMP are organized differently, I am wondering whether this is the main reason for the error. So I attached a screenshot comparing 16S CSV and WIMP CSV as well. 

 
![Screenshot from 2020-11-20 18-29-42](https://user-images.githubusercontent.com/69881217/99772446-e8590380-2b5e-11eb-8d2e-2354ece6d0ad.png)

![Screenshot from 2020-11-20 18-42-05](https://user-images.githubusercontent.com/69881217/99774801-62d75280-2b62-11eb-9112-a0fa6022746f.png)
